spec/models/guest_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
+
+RSpec.describe Guest, type: :model do
+  describe "relationships" do
+    it { should have_many :rooms }
+  end
```

Comment:
I know you're aware of this since we chatted a little after the assessment, but putting this here to keep feedback all in one spot-- make sure to test relationships for GuestRoom, Guests, and Rooms.